packages/lib-classifier/src/store/DrawingStore.spec.js

```diff
@@ -1,41 +1,185 @@
     expect(model.eventStream.subscribe).to.be.a('function')
   })
 
-  it('should add new events to the stream on calling `addToStream`', function (done) {
-    const newEvent = 'foo'
-    model.eventStream.subscribe(value => {
-      expect(value).to.equal(newEvent)
-      done()
+  describe('Views > activeDrawingTask', function () {
+    it('should return the active workflow step drawing task', function () {
+      const drawingTask = DrawingTaskFactory.build()
+      drawingTask.help = 'test help string'
+      const workflow = WorkflowFactory.build({
+        steps: [
+          ['S1', { taskKeys: ['T9'] }]
+        ],
+        tasks: {
+          T9: drawingTask
+        }
+      })
+      const project = ProjectFactory.build({}, { activeWorkflowId: workflow.id })
+      const panoptesClientStub = stubPanoptesJs({ projects: project, workflows: workflow })
+      rootStore = setupStores(panoptesClientStub, project, workflow)
+
+      expect(rootStore.drawing.activeDrawingTask.help).to.eql('test help string')
+      expect(rootStore.drawing.activeDrawingTask.taskKey).to.eql('T9')
     })
-    model.addToStream(newEvent)
   })
 
-  it('should default the active tool index to the first tool in the tools array', function () {
-    expect(model.activeDrawingTool).to.equal(0)
+  describe('Views > isDrawingInActiveWorkflowStep', function () {
+    it('should return true if there is a drawing task in the active workflow step', function () {
+      const workflow = WorkflowFactory.build({
+        steps: [
+          ['S1', { taskKeys: ['T1'] }]
+        ],
+        tasks: {
+          T1: DrawingTaskFactory.build()
+        }
+      })
+      const project = ProjectFactory.build({}, { activeWorkflowId: workflow.id })
+      const panoptesClientStub = stubPanoptesJs({ projects: project, workflows: workflow })
+      rootStore = setupStores(panoptesClientStub, project, workflow)
+
+      expect(rootStore.drawing.isDrawingInActiveWorkflowStep).to.be.true()
+    })
+
+    it('should return false if there is not a drawing task in the active workflow step', function () {
+      const workflow = WorkflowFactory.build({
+        steps: [
+          ['S1', { taskKeys: ['T1'] }]
+        ],
+        tasks: {
+          T1: SingleChoiceTaskFactory.build()
+        }
+      })
+      const project = ProjectFactory.build({}, { activeWorkflowId: workflow.id })
+      const panoptesClientStub = stubPanoptesJs({ projects: project, workflows: workflow })
+      rootStore = setupStores(panoptesClientStub, project, workflow)
+
+      expect(rootStore.drawing.isDrawingInActiveWorkflowStep).to.be.false()
+    })
   })
 
-  it('should set the active tool index', function () {
-    model.setActiveDrawingTool(1)
-    expect(model.activeDrawingTool).to.equal(1)
+  describe('Views > coordinateStream', function () {
+    beforeEach(function () {
+      sinon.stub(model, 'getEventOffset').callsFake((x, y) => ({ x: (x + 100), y: (y + 100) }))
```

Comment:
I plan to revisit stubbing `getEventOffset` with fresh eyes. I briefly attempted to store an `svg` to the DrawingStore and test accordingly without initial success, but might be able to.